.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,57 @@
+name: Bug Report
+description:  Create a report to help us improve
+labels: ["Type: Bug", "Needs: Triage", "Needs: Lead"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **IMPORTANT: Before posting, remove sensitive data, such as passwords, secret keys, session cookies, etc.**
+
+        Thanks for taking the time to fill out this bug report! 👍
+  - type: textarea
+    attributes:
+      label: Problem
```

Comment:
Do we need a label? Will it work if we just don't have one?
```suggestion
```

Reply:
Unfortunately, label is required. https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#textarea